templates/package/content/container.tmpl

```diff
@@ -36,11 +36,13 @@
 				</thead>
 				<tbody>
 					{{range .PackageDescriptor.Metadata.Manifests}}
-					<tr>
-						<td><a href="{{$.PackageDescriptor.PackageWebLink}}/{{PathEscape .Digest}}">{{.Digest}}</a></td>
-						<td>{{.Platform}}</td>
-						<td>{{FileSize .Size}}</td>
-					</tr>
+						{{if ne .Platform "unknown/unknown"}}
+						<tr>
+							<td><a class="ui sha" href="{{$.PackageDescriptor.PackageWebLink}}/{{PathEscape .Digest}}">{{StringUtils.TrimPrefix .Digest "sha256:" | ShortSha}}</a></td>
```

Comment:
There is no styles for selector `.ui.sha`

If you'd like to revert, please do a complete revert.

Reply:
updated: [0a59e1a](https://github.com/go-gitea/gitea/pull/32851/commits/0a59e1afdb7831c2382923b1d58f271612e18dc2)